lsm-tree.cabal

```diff
@@ -121,65 +124,307 @@ description:
 
   (*The variable \(b\) refers to the number of entries retrieved by the range lookup.)
 
-  TODO: Document the average-case behaviour of lookups.
+  === Table Size #performance_size#
 
-  === In-memory size of tables #performance_size#
+  The in-memory and the on-disk size of an LSM-tree scale /linearly/ with the number of physical entries.
+  However, the in-memory size is smaller by a significant factor.
+  Let us look at a table that uses the default configuration and has 100 million entries with 34 byte keys and 60 byte values.
+  The total size of 100 million key–value pairs is approximately 8.75GiB.
+  Hence, the on-disk size would be at least 8.75GiB, not counting the overhead for metadata.
 
-  The in-memory size of an LSM-tree is described in terms of the variable \(n\), which refers to the number of /physical/ database entries.
-  A /physical/ database entry is any key–operation pair, e.g., @Insert k v@ or @Delete k@, whereas a /logical/ database entry is determined by all physical entries with the same key.
+  TODO: Choose default Bloom filter size.
 
-  The worst-case in-memory size of an LSM-tree is \(O(n)\).
+  The in-memory size would be approximately 0.99GiB:
```

Comment:
Maintaining consistency is awkward!

Reply:
I just left my converter on bytes for too long. Or do you mean "repeated text"? Then, yes. If only users only engaged with my docs in the order that I want them to.